src/gui/folderman.cpp

```diff
@@ -1710,8 +1714,7 @@ void FolderMan::trayOverallStatus(const QList<Folder *> &folders,
     }
 }
 
-QString FolderMan::trayTooltipStatusString(
-    SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused)
+QString FolderMan::trayTooltipStatusString(SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused, const ProgressInfo *progress)
```

Comment:
regarding `progress` do you intend the pointer to be constant or the pointed value ?
currently, you declare a `progress pointer to a `const ProgressInfo` value and it looks you mean the opposite
```suggestion
QString FolderMan::trayTooltipStatusString(SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused, ProgressInfo * const progress)
```